.github/ISSUE_TEMPLATE/1_plugin_request.md

```diff
@@ -10,7 +12,7 @@ USE THE TEMPLATE. Otherwise your plugin request may be rejected.
 First, see the contribution guidelines and plugin request requirements:
 https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink
```

Comment:
Should this link be replaced with https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#plugin-requests do you think? I feel like many people still aren't reading the plugin request details.

Reply:
I'm not sure. It currently links to the whole "contributing" section, just like all the other templates do. The plugin requests section is a sub-section, and linking directly to it cuts off the overall issue-reporting guidelines from above. But since it'd link directly to the plugin requirements list, it could improve visibility. This doesn't help though if people don't open the link anyway, and those who do and actually read it will see fewer relevant informations.

Reply:
Yeah fair enough. Maybe a second section that links to it directly calling out the plugin request requirements more clearly?

Reply:
https://github.com/streamlink/streamlink/compare/da3ca957337c448397cab818ef389c34a293ac28..c7673a52e54c19dea3ff06301f8409def5d7759b

Reply:
Looks great!